File: Programmers/level1.dart
class Solution {
  String solution(String s) {
    List<int> characters =
    s.split('').map((title) => title.codeUnitAt(0)).toList();
    characters.sort();
    characters = characters.reversed.toList();
    String answer = '';

    for(var i = 0; i <= s.length - 1; i++) {
      answer += String.fromCharCode(characters[i]);
    }
    return answer;
  }
}

class Solution {
  List<num> solution(int x, int n) {
    List<num> answer = [x];
    for (var i = 2; i <= n; i++) {
      answer.add(answer.first * i);
    }

    return answer;
  }
}

File: Programmers/level1.kt
class Solution {
    fun solution(s: String): String {
        val characters = s.toCharArray()
        characters.sort()
        var answer = ""
        for(c in characters.toList().asReversed()) {
            answer += c
        }
        return answer
    }
}

class Solution {
    fun solution(x: Int, n: Int): LongArray {
        val answer = mutableListOf(x.toLong())
        for (i in 2..n) {
            answer.add(i - 1, answer.first() * i)
        }

        return answer.toLongArray()
    }
}